factory/.env

```diff
@@ -8,7 +8,10 @@
 BASE_IMAGE='debian:bullseye-slim'
 
 # Node Versions: https://nodejs.org/en/download/releases/
-NODE_VERSION='18.14.1'
+DEFAULT_NODE_VERSION='18.15.0'
```

Comment:
Is this the factory node version? If so, maybe rename to: `FACTORY_NODE_VERSION`